types/marked/index.d.mts

```diff
@@ -0,0 +1 @@
+export * from "./index.js";
```

Comment:
what is `index.js` referring to here? should it be `index.d.ts`?

Reply:
Nah - esm imports have to have extensions and TS expects them to be the "output" extensions - so the js file extension.